### PR TITLE
Aws documentations

### PR DESCRIPTION
**Pull request by alero-awani**
**Notes for Reviewers**

- This PR fixes #

- For ui changes: Please add a screenshot showcasing the UI before/after changes. 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

image

### CODE REVIEW

**Comment by Alero-Awani** [comment]
@vishalvivekm Please let me know if I should change the format of this PR

**Comment by l5io** [comment]
🚀 Preview for commit 45db4aefd0c5024afbc380b388fd7f47a6cfe502 at: https://6728a6405e986f531a176376--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3630846aeaa6d65c026f3ab038cb297672d2898a at: https://6728f744dd3a370f11b537f9--meshery-docs-preview.netlify.app

File: docs/_modelscustominfo/aws-ec2.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: AWS EC2
+---
+
+<h1> How to Deploy AWS EC2 Instances with Meshery</h1>
+<p>
+    Learn how to configure, deploy, and manage AWS EC2 instances using Meshery's intuitive <a href="https://docs.meshery.io/extensions/kanvas">Kanvas</a> extension. In order to manage AWS resources, Meshery uses <a href="https://aws.amazon.com/blogs/containers/aws-controllers-for-kubernetes-ack/">AWS Controllers for Kubernetes (ACK)</a>. ACK facilitates the bridge between Kubernetes and AWS services, enabling Meshery to manage AWS resources and Meshery enabling you to benefit from the enhanced experience that Meshery and its extensions offer.
+</p>
+
+<p>
+    To get started, you'll connect your Kubernetes cluster to Meshery. Using pre-configured designs available in the <a href="https://meshery.io/catalog">Meshery Catalog</a>, you'll then configure the EC2 controller with AWS credentials, deploy essential resources (like VPCs, subnets, and security groups) to create the network topology for your EC2 instance. Finally, you'll deploy the instance itself and verify its status through the AWS Management Console or Meshery's Kanvas operator mode.
```
Comment (vishalvivekm) [inline docs/_modelscustominfo/aws-ec2.md:11]:
```suggestion
    To get started, you'll connect your Kubernetes cluster to Meshery. Using pre-configured designs available in the <a href="https://meshery.layer5.io/catalog">Meshery Catalog</a>, you'll then configure the EC2 controller with AWS credentials, deploy essential resources (like VPCs, subnets, and security groups) to create the network topology for your EC2 instance. Finally, you'll deploy the instance itself and verify its status through the AWS Management Console or Meshery's Kanvas operator mode.
```

**Pull request merged**